src/notebooklm/cli/source.py

```
@@ -767,7 +769,7 @@ def source_stale(ctx, source_id, notebook_id, client_auth):
     nb_id = require_notebook(notebook_id)
 
     async def _run():
-        async with NotebookLMClient(client_auth) as client:
+        async with NotebookLMClient(client_auth, timeout=timeout) as client:
```

Ruff (0.15.11)

[error] Undefined name `timeout`

(F821)